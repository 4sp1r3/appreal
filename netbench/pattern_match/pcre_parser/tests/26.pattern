/\x2b\x6Apcre/