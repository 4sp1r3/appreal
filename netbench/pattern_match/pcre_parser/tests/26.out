lex	: modif_front: SLASH
yacc	: pcre_delim <<< SLASH
yacc	: modif_front <<< pcre_delim
lex	: HEX \x2b
yacc	: hex <<< HEX +
yacc	: element <<< hex
yacc	: unit <<< element
lex	: HEX \x6A
yacc	: ext_unit <<< unit
yacc	: hex <<< HEX j
yacc	: element <<< hex
yacc	: unit <<< element
lex	: ASCII p
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII p
yacc	: unit <<< element
lex	: ASCII c
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII c
yacc	: unit <<< element
lex	: ASCII r
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII r
yacc	: unit <<< element
lex	: ASCII e
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII e
yacc	: unit <<< element
lex	: SLASH
yacc	: ext_unit <<< unit
yacc	: exp <<< ext_unit
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: ext_exp <<< exp
yacc	: rv <<< ext_exp
yacc	: inslash <<< rv
yacc	: pattern <<< inslash
yacc	: pcre_delim <<< SLASH
yacc	: modif_rear <<< pcre_delim
yacc	: pcre <<< modif_front pattern modif_rear
